src/Umbraco.Tests.Common/Builders/UserBuilder.cs

```diff
@@ -113,7 +124,7 @@ public UserBuilder<TParent> WithSuffix(string suffix)
 
         public override User Build()
         {
-            var id = _id ?? 1;
+            var id = _id ?? 0;
```

Comment:
This feels intentional but possibly breaking, not sure why it was 1 previously but thought I'd raise in case it's accidental

Reply:
Yes.. But this is just the Test builder. I think it was by mistake 1 before.. All tests updated